tests/015-test-fail-loop-to-create-thin-pool.sh

```diff
@@ -0,0 +1,60 @@
+source $SRCDIR/libtest.sh
+
+test_fail_if_loop_partition_passed() {
+  local lbdevice tmpfile
+  local test_status=1
+  local testname=`basename "$0"`
+  local vg_name="css-test-foo"
+  local tmplog=${WORKDIR}/tmplog
+  local errmsg="Partition specification unsupported at this time."
+
+  # Error out if any pre-existing volume group vg named css-test-foo
+  if vg_exists "$vg_name"; then
+    echo "ERROR: $testname: Volume group $vg_name already exists." >> $LOGS
+    return $test_status
+  fi
+
+  # Create loopback device.
+   tmpfile=$(mktemp /tmp/c-s-s.XXXXXX)
+   truncate --size=6G $tmpfile
+   lbdevice=$(losetup -f)
+   losetup --partscan $lbdevice $tmpfile
+   if ! create_partition $lbdevice;then
+      echo "ERROR: Failed partitioning $lbdevice"
```

Comment:
If we fail to partition the device, we should fail test and return with status 1 here, isn't it?